### DISCUSSION TITLE
Améliorer les statistiques

### DISCUSSION THREAD

**Discussion by johangirod** [publicodes/publicodes#523] [Ideas]
Aujourd'hui la page /stats référence le nombre de visites sur le site publi.codes

Je propose de l'améliorer en l'axant sur l'impact (comme pour les autres produits beta) : 
- nombre de ligne de code (https://github.com/mquandalle/count-publicodes-lines)
- date du dernier commit / de la dernière version
- nombre de téléchargement du paquet
- nombre de paquet de règles publiées
- ...

**Comment by mquandalle** [comment]
L'iframe fonctionne 

![image](https://github.com/publicodes/publicodes/assets/1730702/fc7f1df0-073b-4290-9980-7d5a2bb1eb71)


J'imagine que c'est ublock origin qui la bloque dans ton cas https://github.com/plausible/analytics/issues/2550

Mais d'accord avec l'idée d'y référencer des métriques d'impact.
